spec/_attachments/ic.did

```diff
@@ -186,6 +192,15 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  node_metrics : (record {
+    subnet_id : principal;
+    requested_timestamp_nanos: nat64;
```

Comment:
`start_at_timestamp_nanos` might be more descriptive now that we return all available metrics since this timestamp.

Reply:
Should it maybe also `node_metrics_history` or something similar? If we return a time series, we should somehow make clear that you would receive a time series.

Reply:
Sure, done.